src/core/geometry/poly3/measureSignedVolume.js

```diff
@@ -1,14 +1,12 @@
 const vec3 = require('../../math/vec3')
 
+// NOTE: This is totally worthless. The volume of a planar polygon is always zero(0).
 const measureSignedVolume = poly3 => {
```

Comment:
hmm very good point, but was perhaps the original intention of this to measure the signed volume OCCUPIED by the polygon ? ie the bounding box volume ?